tests/testUtil.ts

```diff
@@ -134,6 +135,29 @@ export const createDynamoDbStreamEvent = (
     })),
 })
 
+export const createKinesisStreamEvent = (
+    ...tableArns: string[]
+): KinesisStreamEvent => ({
+    Records: tableArns.map((tableArn) => ({
+        kinesis: {
+            partitionKey: "partitionKey-03",
+            kinesisSchemaVersion: "1.0",
+            data: "SGVsbG8sIHRoaXMgaXMgYSB0ZXN0IDEyMy4=",
+            sequenceNumber:
+                "49545115243490985018280067714973144582180062593244200961",
+            approximateArrivalTimestamp: 1428537600,
+        },
+        eventSource: "aws:kinesis",
+        eventID:
+            "shardId-000000000000:49545115243490985018280067714973144582180062593244200961",
+        invokeIdentityArn: "arn:aws:iam::EXAMPLE",
+        eventVersion: "1.0",
+        eventName: "aws:kinesis:record",
+        eventSourceARN: `${tableArn}`,
+        awsRegion: "us-east-1",
+    })),
+})
+
```

Comment:
Is the data in this test event sensitive in any way?  It looks like it might be copied from a real event.

Reply:
Hi @johnmikk . the data was copied from test data generated in AWS console. Is not a real data